tests/rkt-monitor/main.go

```diff
@@ -40,9 +43,11 @@ type ProcessStatus struct {
 var (
 	pidMap map[int32]*process.Process
 
-	flagVerbose    bool
-	flagDuration   string
-	flagShowOutput bool
+	flagVerbose      bool
+	flagDuration     string
+	flagShowOutput   bool
+	saveToCsv        bool
+	repetitionNumber int
```

Comment:
Maybe prepend these added variables with `flag` to be consistent with the other flag variables?